docs/Makefile

```diff
@@ -76,3 +76,9 @@ container-server: container-serve
 
 clean: ## Delete generated content. (requires sudo if using container-serve)
 	rm -rf public resources node_modules public.zip
+
+compute-checksum: ## Compute docs checksum
+	sha256sum public/index.html > DOCS_CHECKSUM
+
+upload-checksum: compute-checksum ## Upload the checksum to artifacts bucket
+	aws s3 cp DOCS_CHECKSUM s3://artifactsstack-1532887287-artifactsbucket2aac5544-exgxrmntr4id --acl public-read
```

Comment:
We shouldn't hardcode this here. We should set it as an env var from CDK.

Reply:
Updated